src/lib/answer-config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  noneOfTheAbove: {
+    key: "something.noneOfTheAbove",
```

Comment:
are these supposed to `something` Looking at the mappings below in the locales, should it not be `answers.x`

Reply:
Oh yeah, that is such draft code - and I'm not sure this would/should work with browser tests either.

I'll update this PR to more properly work :)